docs/automatic-release.md

```diff
@@ -40,9 +40,10 @@ jobs:
 
 #### Inputs
 
-| Name           | Default | Description                                          |
-|----------------|---------|------------------------------------------------------|
-| `NODE_VERSION` | 18      | Node version with which the release will be executed |
+| Name           | Required | Default | Description                                                                                |
+|----------------|----------|------------------|--------------------------------------------------------------------------------------------|
+| `NODE_VERSION` | false    |18               | Node version with which the release will be executed                                       |
+| `USER_TOKEN`   | false    |''          | Authentication token with write permission needed by the release bot. When not defined will fallback to GITHUB_TOKEN |
```

Comment:
```suggestion
| `ACCESS_TOKEN`   | false    |''          | Authentication token with write permission needed by the release bot (falls back to GITHUB_TOKEN) |
```
Not sure about `ACCESS_TOKEN` or if we just should also use `GITHUB_USER_TOKEN` in the documentation.

Also, please update the Markdown table structure. Thanks!